docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,86 @@
+
+> [!NOTE]
+> The .NET API analyzer is still a pre-release version.
+
+## Discovering deprecated APIs
+
+**What are deprecated APIs?**
+
+The .NET family is a set of large products that are constantly upgraded to better serve customer needs. It is natural to deprecate some APIs and replace them with new ones. One way to inform that an API is deprecated and should not be used is to mark it with the `[Obsolete]` attribute. The disadvantage of this approach is that there is only one diagnostic ID for all obsolete APIs (for C#, [CS0612](../../csharp/misc/cs0612.md)) that leads to one description for all cases and ability to suppress warnings for either all cases or none of them. The API Analyzer uses API-specific error codes that begin with DE (which stands for Deprecation Error), which allows control over the display of individual warnings. To avoid confusion with the `[Obsolete]` attribute, this concept is called deprecation.
+
+When a deprecated API, such as <xref:System.Net.WebClient>, is used in a code, API Analyzer highlights it with a green squiggly line and shows a light bulb on the left as in the following example:
+
+!["Screenshot of WebClient API with green squiggly line and light bulb on the left"](media/api-analyzer/green-squiggle.jpg)
+
+The **Error List** window contains warnings with a unique ID per deprecated API, as shown in the following example (`DE004`): 
+
+!["Screenshot of the Error List window showing warning's ID and description"](media/api-analyzer/warnings.jpg)
+
+By clicking on the ID, you go to a webpage with detailed information about why the API was deprecated and suggestions regarding alternative APIs that can be used.
+
+Any warnings can be suppressed by right-clicking on the highlighted member and selecting **Quick Actions and Refactorings**. There are two ways to supress warnings: 
+
+* locally (in source)
+* globally (in a suppression file)
+
+We encourage developers to use global suppression to ensure consistency of API usage across their projects.
+
+## Discovering cross-platform issues
+
+Similar to deprecated APIs, the analyzer identifies all APIs that are not cross-platform supported. For example, <xref:System.Console.WindowWidth?displayProperty=nameWithType> works on Windows but not on Linux and macOS. The diagnostic ID is shown in **Error List** window. You can suppress that warning by right clicking and choosing **Quick Actions and Refactorings**. Unlike deprecation cases where you have two options (either keep using deprecated member and suppress warnings or not use it at all), here if you're developing your code only for certain platform you can suppress all warnings for all other platforms you don't plan to run your code on. To do so, you just need to edit your project file and add the `PlatformCompatIgnore` property that lists all platforms to be ignored. The accepted values are: `Linux`, `MacOSX`, and `Windows`.
```

Comment:
cross-platform supported --> supported cross-platform
in \*\*Error List\*\* --> in the \*\*Error List\*\*
using deprecated --> using the deprecated
for certain platform --> for certain platforms, (not comma after platforms)